plasmapy/diagnostics/thomson.py

```diff
@@ -250,6 +257,13 @@ def spectral_density_lite(
     # Apply an instrument function if one is provided
     if instr_func_arr is not None:
         Skw = np.convolve(Skw, instr_func_arr, mode="same")
+
+    # add notch(es) to the spectrum if any are provided
```

Comment:
Subtle point - do we want to apply the notch before or after the instrument function? I think before - otherwise the IRF will convolve data from inside the notch region into the Skw on the edges of the notch, but that data doesn't exist in the experiment because the notch filter probably comes before the spectrometer slit? Check with Derek on this one, correct the order if necessary and add a comment clarifying why this is important (if it is) since it's subtle and we don't want it changed back!